streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBStoreTest.java

```diff
@@ -402,6 +402,11 @@ public void shouldReturnKeysWithGivenPrefix() {
         assertThat(valuesWithPrefix.get(2), is("b"));
     }
 
+    @Test
+    public void shouldThrowNullPointerIfPrefixKeySerializerIsNull() {
```

Comment:
This test fails now, as the check was removed. No need to add this test any longer in this class.